packages/massa-web3/src/experimental/account.ts

```diff
@@ -0,0 +1,342 @@
+import * as ed from '@noble/ed25519'
+import { base58Decode, hashBlake3, varintDecode } from '../utils/Xbqcrypto'
+
+const SECRET_KEY_PREFIX = 'S'
+const PUBLIC_KEY_PREFIX = 'P'
+const ADDRESS_PREFIX = 'A'
+const ADDRESS_USER_PREFIX = 'U'
+const ADDRESS_CONTRACT_PREFIX = 'S'
+
+export enum Version {
+  V0 = 0,
+}
+
+export class SecretKey {
+  private bytes: Uint8Array
+  public version: Version
+
+  private constructor(bytes: Uint8Array, version: Version) {
+    this.bytes = bytes
+    this.version = version
+  }
```

Comment:
This is a duplicate of SecretKey.fromBytes. Furthermore, why not injecting the following dependencies instead:
- string format (encode, decode of base58 for instance)
- crypto functions (sign, generateKeys, hash)

the probability that we change the crypto primitives is low, but non null. injecting dependencies will make this modification smoother and will make unit test coverage easier.